parser.y

```diff
@@ -2100,6 +2101,20 @@ ColumnOption:
 	{
 		$$ = &ast.ColumnOption{Tp: ast.ColumnOptionCollate, StrValue: $2.(string)}
 	}
+|   "COLUMN_FORMAT" "DEFAULT"
+	{
+		$$ = &ast.ColumnOption{Tp: ast.ColumnOptionColumnFormat, StrValue: "DEFAULT"}
+	}
+|
+    "COLUMN_FORMAT" "FIXED"
+	{
+		$$ = &ast.ColumnOption{Tp: ast.ColumnOptionColumnFormat, StrValue: "FIXED"}
+	}
+|
+    "COLUMN_FORMAT" "DYNAMIC"
+	{
+		$$ = &ast.ColumnOption{Tp: ast.ColumnOptionColumnFormat, StrValue: "DYNAMIC"}
+	}
```

Comment:
It would be better to abstract `DEFAULT`, `FIXED`, `DYNAMIC` as a new production named `columnFormat`.

Reply:
What's the benefit to create new columnFormat name? Can it be reused by anywhere else?

Reply:
No, it can't. Just in line with the existing style.

Reply:
got you, will do.